example-map.js

```diff
@@ -0,0 +1,14 @@
+const exampleMap = {
+    "Language": {
+      fields: {
+        id: "builtin-pt, builtin-de, builtin-es",
```

Comment:
Think all the fields here should only show one value in the response, e.g., you wouldn't get a response where the `id` was `builtin-pt, builtin-de, builtin-es`, you'd only get `code: 'builtin-pt'`. So for the field values, we can pick just one value of the three and use that instead.